plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2420,8 +2423,8 @@ public Optional<ConnectorViewDefinition> getView(ConnectorSession session, Schem
                         throw new HiveViewNotSupportedException(viewName);
                     }
 
-                    ConnectorViewDefinition definition = createViewReader(metastore, session, view, typeManager, this::redirectTable, metadataProvider)
-                            .decodeViewData(view.getViewOriginalText().get(), view, catalogName);
+                    ConnectorViewDefinition definition = createViewReader(metastore, session, view, typeManager, this::redirectTable, metadataProvider, true)
```

Comment:
why does this have a hard coded `true` but the next line is configurable?  If this is correct, add a comment explaining what is happening here

Reply:
Also, this appears to be the only caller of this in the PR, and it is hard coded to `true`.  Does this needt to be a parameter at all?

Reply:
Had the wrong approach; re-wrote this based on findepi's guidance